### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
card components added

### CODE REVIEW

File: components/card.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    justify-content: center;
+    align-items: center;
+    width: 15rem;
+    height: 20rem;
+    border-radius: 10px;
+    background: var(--primary-font-colour);
+    box-shadow: 4px 4px 15px var(--box-shadow);
+    position : relative;
+    
+    margin: 10px;
```
Comment (deekshithmd) [inline components/card.css]: giving margin for component will give problem in aligning when you use it in other projects

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+     position: absolute;
+     margin: 10px;
+     padding: 10px;
+    
+     }
+     .hz-heading{
+       font-size: 20px;
+       
+     }
+     .hz-subhead{
```
Comment (deekshithmd) [inline components/card.css:146]: remove extra white spaces and give 1 line gap between each class

File: index.html [hunk 3]
```diff
@@ -333,23 +333,203 @@ <h1>Icon Buttons</h1>
...
+    
+    <div class="container">
+      <h1>Vertical cards</h1>
+      <p>Here we created cards.</p>
+      <p>
+        The card component with simple .
+      </p>
+      <div class="vertical-card">
+        <div class="card-simple">
+          <div class="card__container">
```
Comment (deekshithmd) [inline index.html:378]: avoid using unnecessary div's if possible

File: index.html [hunk 4]
```diff
@@ -333,23 +333,203 @@ <h1>Icon Buttons</h1>
...
+          </p>  
+          </div>
+        </div>
+        </div>
+      </div>
 
+      </div>
 
+    <div class="code-container">
+      <iframe
```
Comment (deekshithmd) [inline index.html:393]: customize the iframe so that it will have same width and height throughout the doc, it makes ui good

File: index.html [hunk 5]
```diff
@@ -333,23 +333,203 @@ <h1>Icon Buttons</h1>
...
+            <img src="https://m.media-amazon.com/images/I/71P5lELL6lL._AC_SR180,120_QL70_.jpg" alt="">
+            <div class="hz-head">
+            <p class="hz-heading">speed hybrid bicyle </p>
+            <div class="hz-subhead"><small>Hero sprint from hero</small></div>
+            <div class="hz-details">
+              <span class="price-discount">&#8377 16499 <span class="price-originl">&#8377 17000</span><span class="offer">(3%OFF)</span> </span>
+              <div class="cart-button"><button><i class="fas fa-shopping-cart"></i>Add to cart</button></div>
+             
+            </div>
+
```
Comment (deekshithmd) [inline index.html:447]: eleminate these kind extra spaces

**Comment by BALAJI141031** [comment]
you can give background to out of stock text for me this text is colliding with background text

File: components/card.css [hunk 6]
```diff
@@ -0,0 +1,131 @@
+.vertical-card{
```
Comment (PuneetP16) [inline components/card.css:1]: You could name the class as "card-vertical" to be consistent with how you are naming classes.

File: components/card.css (hunk 1 above)
Comment (PuneetP16) [reply components/card.css]: yeah, avoid giving margins to components. Instead, you could use flex to layout components.

**Review by PuneetP16 — APPROVED** [review]

File: components/card.css (hunk 1 above)
Comment (SumitSaundadkar) [reply components/card.css]: yes did it

File: components/card.css (hunk 2 above)
Comment (SumitSaundadkar) [reply components/card.css:146]: done

**Review by horsemaker — COMMENTED** [review]
Looks good! I am just worried about class names. I guess you could work on them later to make them more relative & generic.

**Review by saurabhchirde — COMMENTED** [review]
Looks good to merge.

**Comment by TusharJade** [comment]
looks good to me👍

**Pull request merged**